.pre-commit-hooks.yaml

```diff
@@ -35,4 +35,13 @@
   pass_filenames: true
   minimum_pre_commit_version: 0.15.0
   exclude: 'vendor/'
-  description: "Checks changes contain only TODOs with JIRA tags"
\ No newline at end of file
+  description: "Checks changes contain only TODOs with JIRA tags"
+
+- id: branch-name-check
+  name: 'branch name begin with (feature|bugfix|release|hotfix)'
+  entry: run-branch-name-check.sh
+  types: [text]
+  language: 'script'
+  pass_filenames: true
```

Comment:
If we swap:

```yaml
types: [text]
```

which makes the hook run only when adding a text file (excludes binaries, images, etc), and add in it's place:

```yaml
always_run: true
```

The hook will run always 👍 We can also drop `pass_filenames` as the script doesn't need them.